tests/Test.ADAL.NET.Unit/JsonWebTokenTests.cs

```diff
@@ -122,11 +122,21 @@ public void Initialize()
             AuthenticationResult result = await context.AcquireTokenAsync(TestConstants.DefaultResource, clientAssertion, true).ConfigureAwait(false);
             Assert.IsNotNull(result.AccessToken);
 
+            HttpMessageHandlerFactory.AddMockHandler(X5CMockHandler);
+            result = await context.AcquireTokenByAuthorizationCodeAsync(TestConstants.DefaultAuthorizationCode, TestConstants.DefaultRedirectUri, clientAssertion, TestConstants.DefaultResource,
+                true).ConfigureAwait(false);
```

Comment:
> true) [](start = 15, length = 6)

minor: having this on a new line is odd... imo. Consider either splitting all params on new lines or putting this one line above...